DOC/conf.py

```diff
@@ -71,11 +82,12 @@
 
 # General information about the project.
 project = 'tick'
-copyright = '2015, Emmanuel Bacry, Martin Bompaire, Stephane Gaiffas, ' \
+copyright = 'Emmanuel Bacry, Martin Bompaire, Stephane Gaiffas, ' \
```

Comment:
http://ben.balter.com/2015/06/03/copyright-notices-for-websites-and-open-source-projects/#the-copyright-holder

It seems that we are supposed to set it to the copyright holder. Is it Ecole polytechnique in our case?